compiler/compiler/src/main/java/org/robovm/compiler/plugin/shadowframe/ShadowFramePlugin.java

```diff
@@ -40,28 +41,25 @@ public void afterMethod(Config config, Clazz clazz, SootMethod method, ModuleBui
         if (method.isNative() || method.isAbstract() || !method.hasActiveBody()) {
             return;
         }
-
+        
         // insert generation of shadow frame and wiring it up to previously set
         // shadow frame in the entry basic block
         BasicBlock entryBlock = function.getBasicBlocks().get(0);
         Variable shadowVariable = function.newVariable(SHADOW_FRAME_VAR_NAME, Types.SHADOW_FRAME_PTR);
         Alloca shadowAlloca = new Alloca(shadowVariable, Types.SHADOW_FRAME);
         entryBlock.getInstructions().add(0, shadowAlloca);
```

Comment:
I meant getting rid of this alloca of the shadow frame, currently still being allocated on the stack though not used. We can keep the llvm methods in the header in for now.